src/components/phone/CallerIDEditCard.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div>
+    <base-button
+      variant="outline"
+      :action="
+        () => {
+          isShowingModal = true;
```

Comment:
same thing as earlier,
replace with `() => (isShowingModal = true)`

its not wrong, it just is not needed and breaks the syntax highlighting :man_shrugging: